src/game.zig

```diff
@@ -781,8 +781,8 @@ pub fn update(deltaTime: f64) void { // MARK: update()
 			if (Player.onGround) { // No view bobbing in the air
 				Player.bobTime += Player.bobVel * 8 * deltaTime;
 			}
-			// Maximum magnitude when sprinting (2x walking speed)
-			Player.bobMag = @sqrt(@min(Player.bobVel, 2)) * settings.viewBobStrength;
+			// Maximum magnitude when sprinting (2x walking speed). Magnitude is scaled by player bounding box height
+			Player.bobMag = @sqrt(@min(Player.bobVel, 2)) * settings.viewBobStrength * Player.outerBoundingBoxExtent[2];
```

Comment:
I'm still confused about what you mean about the eyeBox in relation to view bobbing. Scaling by the bounding box height should scale the view bobbing by player height, so scaling the player up or down should cause the view bobbing to appear the same relative to the player's size. Is there a plan to have characters with tiny legs and a huge hat or something, where the eyes and view bobbing should be ow but the bounding box tall?

Reply:
Scaling the bobbing by player height is fine, but
Let's say you sprint jump through a narrow tunnel.
I need you to guarantee that under no circumstances the camera will clip into the terrain.
The camera can be anywhere inside the eyeBox The eyeBox should always be smaller than the collision box. But your code adds a value to that cameraPosition, so you might push it outside the eyeBox.
Please ensure that the resulting position is never outside the collision box, which could make it clip into the terrain. It probably shouldn't even get too close to the collision, because OpenGL doesn't render triangles at depth 0.

Now you have probably tested this scenario and everything works fine, but all these parameters, like the hitboxes and how often the camera actually gets close to the bounds of the eyeBox, may change.